db/schema.rb

```diff
@@ -353,6 +353,11 @@
     t.datetime "completed_at"
   end
 
+  create_table "user_profiles", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
It didn't get updated automatically, is there a command we have to run for this?